omim2obo/main.py

```diff
@@ -270,32 +278,45 @@ def omim2obo(use_cache: bool = False):
             if len(assocs) == 1:
                 predicate = MORBIDMAP_PHENOTYPE_MAPPING_KEY_PREDICATES[p_map_key]
 
+            evidence = Literal(f'Evidence: ({p_map_key}) {p_map_lab}')
+            
             # i. Add to MIM class
             # noinspection DuplicatedCode  b_and_b3_diff_results
             b = BNode()
             graph.add((b, RDF['type'], OWL['Restriction']))
             graph.add((b, OWL['onProperty'], predicate))
-            if p_mim:
-                graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
+            graph.add((b, OWL['someValuesFrom'], OMIM[p_mim]))
             graph.add((OMIM[mim_number], RDFS['subClassOf'], b))
 
             # ii. add axiom
             b2 = BNode()
             graph.add((b2, RDF['type'], OWL['Axiom']))
             graph.add((b2, OWL['annotatedSource'], OMIM[mim_number]))
             graph.add((b2, OWL['annotatedProperty'], RDFS['subClassOf']))
-            # todo: add mapping key and related info in a different way?
-            evidence = Literal(f'Evidence: ({p_map_key}) {p_map_lab}')
-            # todo: add biolink:GeneDiseaseAssociation structure as well?
+            graph.add((b2, OWL['annotatedTarget'], b))
             graph.add((b2, BIOLINK['has_evidence'], evidence))
             graph.add((b2, RDFS['comment'], evidence))
-            # noinspection DuplicatedCode b_and_b3_diff_results
-            b3 = BNode()
-            graph.add((b3, RDF['type'], OWL['Restriction']))
-            graph.add((b3, OWL['onProperty'], predicate))
-            if p_mim:
-                graph.add((b3, OWL['someValuesFrom'], OMIM[p_mim]))
-            graph.add((b2, OWL['annotatedTarget'], b3))
```

Comment:
`b3` not needed as its exactly the same as `b`.

Reply:
@matentzn Hmm, it's too seldom that I lay with these bnodes in `rdflib`, but I wonder if you are 100% certain of that. See this comment that I left:
> `# noinspection DuplicatedCode b_and_b3_diff_results`

PyCharm had detected that the declarations for `b` and `b3` were duplicated, but `b_and_b3_diff_results` is my personal note, indicating that I at least thought I was getting different results and that doing it this way was correct.

Reply:
Yes, its not identical. There is a bit  of a difference in the turtle serialisation, because the blank nodes gets associated an actual identifier, and in your way of doing things, both expressions had _different_ identifiers, why if you use the same blank node, both expressions get associated with the same blank node. This looks pretty ugly in turtle syntax, but you cant see it in OWL (functional or owl/xml syntax).

Reply:
Hmm I see. If that's the only difference, then it is benign, and it seems your change is an improvement.